modules/nw-multus-ipam-object.adoc

```diff
@@ -240,3 +240,61 @@ Whereabouts can be used for both IPv4 and IPv6 addresses.
 ifdef::sr-iov[]
 :!sr-iov:
 endif::[]
+
+[id="nw-multus-creating-whereabouts-reconciler-daemon-set_{context}"]
+== Creating a Whereabouts reconciler daemon set 
+
+The Whereabouts reconciler is responsible for managing dynamic IP address assignments for the pods within a cluster using the Whereabouts IP Address Management (IPAM) solution. It ensures that each pods gets a unique IP address from the specified IP address range. It also handles IP address releases when pods are deleted or scaled down.
+
+[NOTE]
+====
+You can also use a `NetworkAttachmentDefinition` custom resource for dynamic IP address assignment. 
+====
+
+The Whereabouts reconciler daemon set is automatically created when you configure an additional network through the Cluster Network Operator. It is not automatically created when you configure an additional network from a YAML manifest. 
+
+To trigger the deployment of the Whereabouts reconciler daemonset, you must manually create a `whereabouts-shim` network attachment by editing the Cluster Network Operator custom resource file.
+
+Use the following procedure to deploy the Whereabouts reconciler daemonset. 
+
+.Procedure
+
+. Edit the `Network.operator.openshift.io` custom resource (CR) by running the following command:
++
+[source,terminal]
+----
+$ oc edit network.operator.openshift.io cluster
+----
+
+. Modify the `additionalNetworks` parameter in the CR to add the `whereabouts-shim` network attachment definition. For example:
++
+[source,yaml]
+----
+apiVersion: operator.openshift.io/v1
+kind: Network
+metadata:
+  name: cluster
+spec:
+  additionalNetworks:
+  - name: whereabouts-shim
+    namespace: default
+    rawCNIConfig: |-
+      {
+       "name": "whereabouts-shim",
+       "cniVersion": "0.3.1",
+       "type": "bridge",
+       "ipam": {
+         "type": "whereabouts"
+       }
+      }
+    type: Raw
+----
+
+. Save the file and exit the text editor.
+
+. Enter the following command to apply the `whereabouts-shim` to your Cluster Network Operator configuration:
```

Comment:
I do not think you need step 4 any more after you using step 1

Reply:
For step 4, you can add verification step:
`[weliang@weliang openshift-tests-private]$ oc get all -n openshift-multus | grep whereabouts-reconciler
pod/whereabouts-reconciler-jnp6g                   1/1     Running   0          6s
pod/whereabouts-reconciler-k76gg                   1/1     Running   0          6s
pod/whereabouts-reconciler-k86t9                   1/1     Running   0          6s
pod/whereabouts-reconciler-p4sxw                   1/1     Running   0          6s
pod/whereabouts-reconciler-rvfdv                   1/1     Running   0          6s
pod/whereabouts-reconciler-svzw9                   1/1     Running   0          6s
daemonset.apps/whereabouts-reconciler          6         6         6       6            6           kubernetes.io/os=linux   6s
[weliang@weliang openshift-tests-private]$ 
`

Reply:
@weliang1, I like this, thanks!

Reply:
@weliang1 @jab-rh I think we're finally good here. Thanks for the review!